front/components/workspace/ChangeMemberModal.tsx

```diff
@@ -58,7 +54,7 @@ export function ChangeMemberModal({
           role: selectedRole,
           sendNotification,
         });
-        await mutate(`/api/w/${owner.sId}/members`);
+        await mutateMembers();
```

Comment:
It would be great to convert `handleMembersRoleChange` as a hook so you can have the mutation within it (see `useCreateVault` for example).

Reply:
Yep! Won't have time to handle it now though, got too many runner cards this morning 🏃🏻